configsuite/schema.py

```diff
@@ -267,16 +268,16 @@ def gen_err_msg(error):
 
             if isinstance(error, configsuite.MissingKeyError):
                 raise KeyError(gen_err_msg(error))
-            elif isinstance(error, configsuite.UnknownKeyError):
+            if isinstance(error, configsuite.UnknownKeyError):
                 raise KeyError(gen_err_msg(error))
-            elif isinstance(error, configsuite.InvalidTypeError):
+            if isinstance(error, configsuite.InvalidTypeError):
                 raise TypeError(gen_err_msg(error))
-            elif isinstance(error, configsuite.InvalidValueError):
+            if isinstance(error, configsuite.InvalidValueError):
                 raise ValueError(gen_err_msg(error))
-            else:
```

Comment:
All the `else` statements are leaving 😭 I must admit that I kind of like the `else`-statement in the situations where one is intending to cover "all the possible cases", as it makes it really easy to spot whether there is an `else` and make sure it contains something else then `pass` (or similar).

But I also agree to follow the `pylint` recommendations; and perhaps I like that even more 🤷

Reply:
Yes, that's always a difficult trade-off ... and that's what you get when you let a machine evaluate the code quality, I guess ... the alternative is filling the code base up with `  # noqa` comments. :| 

I don't know what's best, though.